src/main/java/com/znsio/e2e/runner/Runner.java

```diff
@@ -219,4 +219,8 @@ public static String getProxyURL () {
     public static boolean shouldMaximizeBrowser () {
         return configsBoolean.get(BROWSER_MAXIMIZE);
     }
+
+    public static boolean acceptInsecureCerts() {
```

Comment:
rename method to "shouldAcceptInsecureCerts"

Reply:
changes done